roles/tang/templates/tang_override.conf.j2

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Tang Server socket
+Requires=tangd-keygen.service
+Requires=tangd-update.service
+Requires=tangd-update.path
+After=tangd-keygen.service
+After=tangd-update.service
+
+[Socket]
+{%- if tang_port is string %}
```

Comment:
The default value is an int and not a string, not sure if this will work.

Also, since you have a default value of 80, I don't see why this conditional should be necessary at all.